src/SassBuilder.php

```diff
@@ -26,24 +26,26 @@ public function __construct(
         private readonly string $projectRootDir,
         private readonly ?string $binaryPath,
         private readonly bool $embedSourcemap,
+        private readonly ?array $sassOptions = [],
```

Comment:
why nullable ?

Reply:
Changed for a single nullable argument... don't know if this is perfect on a BC pov .. :|